templates/emails/onboarding.day2.hbs

```diff
@@ -0,0 +1,21 @@
+Subject: Give visibility to your backers and sponsors
+
+{{> header}}
+
+<h1>Hey {{collective.name}} 👋</h1>
+
+<p>I’m Pía, one of the founders of Open Collective. I’d love to help you set up and get your collective up and running. Feel free to reach out to me directly replying to this email. I'm here to help you!</p>
+
+<p>Our main piece of advice is this: get your first contributors now! Reach out to your friends & family for them to give you that first boost, even if it’s just for a couple of months. Having those first backers makes a big difference, don’t be shy!</p>
+
+<p>Here is a helpful guide: <a href="https://medium.com/open-collective/ten-steps-to-successful-open-source-crowdfunding-fa2b43e82687">Ten Steps to Successful Open Source Crowdfunding</a></p>
```

Comment:
This  link was a button on the emails I was sending. Is it one here? I think it makes sense.

Reply:
It's quite a very long label for a button.
Also, shouldn't we send this only to open source collectives?

Reply:
I had the button: Get Started! (with the link). 
yes, why do you think it's related?

Reply:
Oh I thought you wanted the link to the 10 steps for successful open source crowdfunding as a button.

So what should get started link to? Simply to the collective?

Like this?

![](https://d.pr/i/WwPxd+)

Reply:
I had the Get started linked to the 10 steps: 
<img width="600" alt="screen shot 2018-01-17 at 5 31 20 pm" src="https://user-images.githubusercontent.com/3671070/35065541-46ee070a-fbac-11e7-9d27-8f85342403d3.png">
It's up to you really, I think buttons are good CTAs. 2cents. don't block on this.